src/components/pages/permissions.tsx

```diff
@@ -19,100 +21,78 @@ function Permissions(): JSX.Element {
-        </div>
-        <h2 className="heading-1">
-          The first letter (red) indicates the file type.
-        </h2>
-        <div className="body">
-          Lorem <span className="try-out-command">d</span> dolor sit amet,
-          consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
-          labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
-          exercitation
-        </div>
-        <h2 className="heading-1">
-          The first letter (red) indicates the file type.
-        </h2>
-        <div className="body">
-          Lorem <span className="try-out-command">d</span> dolor sit amet,
-          consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
-          labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
-          exercitation
-        </div>
-        <h2 className="heading-1">
-          The first letter (red) indicates the file type.
-        </h2>
-        <div className="body">
-          Lorem <span className="try-out-command">d</span> dolor sit amet,
-          consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
-          labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
-          exercitation
-        </div>
-        <h2 className="heading-1">Subsection</h2>
-        <p className="body">
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
-          minim veniam, quis nostrud exercitation
-        </p>
-
-        <div
-          style={{
-            background: 'gray',
-            width: 200,
-            height: 50,
-            marginLeft: 'auto',
-            marginRight: 'auto',
-          }}
-        >
-          placeholder
-        </div>
-
-        <p className="body">
-          Lorem <span className="try-out-command">command</span>
-          dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
-          quis nostrud exercitation
-        </p>
-
-        <div
-          style={{
-            background: 'gray',
-            width: 200,
-            height: 50,
-            marginLeft: 'auto',
-            marginRight: 'auto',
-          }}
-        >
-          placeholder
+    <>
+      <div>
+        <div className="permissionContainer">
+          <h1 className="lesson-title">Permissions</h1>
+          <div className="body">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat. Duis aute irure dolor in
+            reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+            pariatur.
+          </div>
+          <div
+            style={{
```

Comment:
I would recommend pulling this out of the .tsx file and into the .scss file just for consistency and best practice, but for now it's ok.